ThreadPool/ThreadPool/IMyTask.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace ThreadPool
+{
```

Comment:
```suggestion
namespace ThreadPool;

﻿﻿using System;
```
И дальше без лишнего отступа, здесь и ниже. C# 10 же.